src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -38,7 +38,7 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
 
     function getEditsForAction(context: RefactorContext, actionName: string): RefactorEditInfo | undefined {
         const { file, startPosition } = context;
-        const info = getConvertibleArrowFunctionAtPosition(file, startPosition);
+        const info = getConvertibleArrowFunctionAtPosition(file, startPosition, /*triggerReason*/ { kind: "invoked" });

```

Comment:
This feels very awkward because it's not a request with a trigger reason.  Personally, I would probably change the parameter to a boolean, `isAllowedInFunctionBody`.

Reply:
I agree it does feels awkward. Maybe it makes more sense to have this be `isImplicit` and default to false? That way we can just ignore the parameter when the function is called for `getEditsForAction`.